hr_public_holidays/README.rst

```diff
@@ -32,7 +32,7 @@ public holidays
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/116/8.0
+   :target: https://runbot.odoo-community.org/runbot/116/9.0
 
 Known issues / Roadmap
 ======================
```

Comment:
@rami-wafaie Thanks for porting the module, can you please change the bug tracker link version from 8 to 9.